### ISSUE TITLE
iOS 9 app crash on open

### ISSUE THREAD

**Issue by dylan-chong** [mattermost/mattermost-mobile#1988]
Submit feature requests to http://www.mattermost.org/feature-requests/. File non-security related bugs here in the following format: 

#### Summary
Issue in one concise sentence.

#### Environment Information

- Device Name: iPhone 5s
- OS Version: 9.3.3
- Mattermost App Version: 1.10.0
- Mattermost Server Version: not relevant since I can't login 

#### Steps to reproduce

Download the app on an iOS 9 device
Open it
Crash

#### Expected behavior
Don't crash

#### Observed behavior

```
Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Triggered by Thread:  3

```

Please see your iTunes connect  crash log because there maybe personal info on the rest of the crash report

#### Possible fixes
If you can, link to the line of code that might be responsible for the problem.

**Comment by amyblais** [comment]
Hi @dylan-chong! Thanks for reaching out.

@enahum do you have access to the full iTunes crash log?

**Comment by enahum** [comment]
I do, but there is nothing in there for an iPhone 5s iOS 9.3.3, at least not for the 1.10.0 version.

**Comment by amyblais** [comment]
@dylan-chong Are you using build version 127?

**Comment by dylan-chong** [comment]
I am using the latest version from the App Store 

By the way, I've just disabled and re enabled the 'share with app developers' privacy setting, so hopefully that helps it to show up now  on iTunes connect

**Comment by amyblais** [comment]
@enahum Do you see crash reports for this now?

**Comment by enahum** [comment]
Yes I got the report now.. we can take a closer look but it seems to be a Native RN causing the crash, we could look into it to see if we can fix it although I would recommend updating the OS

**Comment by sjstyle** [comment]
My teammate experienced same problem, and I fixed it. (It was crashed on iPhone 6, iOS 9.3.5)
I will create PR if you don't mind.

**Comment by enahum** [comment]
@sjstyle please submit the PR and mention me once you do so I can review it, there are a couple more days to cut the next release so having this soon will help.

Thank you

**Comment by lindalumitchell** [comment]
I just did some testing on a device on iOS 9.3.5, on build 132 of the app, and no crashing so far. 👍
